explore/src/main/scala/explore/Explore.scala

```diff
@@ -170,7 +170,12 @@ object ExploreMain extends IOApp.Simple {
           _                    <- logger.info(s"Config: ${appConfig.show}")
           ctx                  <-
             AppContext.from[IO](appConfig, reconnectionStrategy, pageUrl, setPage)
-          r                    <- (ctx.sso.whoami, setupDOM(), showEnvironment(appConfig.environment)).parTupled
+          r                    <- (ctx.sso.whoami.handleError(_ =>
+                                     none
+                                   ), // recover to enable guest users on unauthorized domains
+                                   setupDOM(),
+                                   showEnvironment(appConfig.environment)
+                                  ).parTupled
```

Comment:
Does this work? Isn't the problem with SSO authorization?

Reply:
So, we cannot auth because of the domain but this change will capture the error and set auth to none, so you get the login view.
Still now testing even guest isn't working, I need to do some extras I think

Reply:
What happens if there's a legit error calling SSO? (It's down for example).